app/controllers/plan_exports_controller.rb

```diff
@@ -34,7 +34,7 @@ def show
     @hash           = @plan.as_pdf(current_user, @show_coversheet)
     @formatting     = export_params[:formatting] || @plan.settings(:export).formatting
     
-    if params.key?(:phase_id)
+    if params.key?(:phase_id) && params[:phase_id].length > 0
```

Comment:
Is this left over from previous work from the multiphase download?

Reply:
yes. This change is necessary to make muti-phase download work. I found it in testing muti-phase download after node-sass upgrade